src/gis-utils/components/ol-geolocate.ts

```diff
@@ -80,4 +82,21 @@ export class Geolocate extends ol.control.Control {
     );
     return feature;
   }
+
+  private _addPositionFeature(pos, view, source) {
+    const zoomLevel = this.options.zoomLevel ? this.options.zoomLevel : 12;
+
+    const positionFeature = this._createFeature();
```

Comment:
ga je hier niet heel veel features aanmaken in dezelfde laag? miss beter eens de laag leeg maken, of de bestaande feature aanpassen.